website/docs/plugin/log/writing.mdx

```diff
@@ -21,7 +21,7 @@ One of the following SDK versions is required to properly output structured logs
 - [`terraform-plugin-framework`](/terraform/plugin/framework) version 0.6.0 or later
 - [`terraform-plugin-sdk`](/terraform/plugin/sdkv2) version 2.10.0 or later
 
-Follow the [legacy logging](#legacy-logging) instructions to write logs with older versions.
+Follow the [legacy logging](#legacy-logging) instructions to write logs with older versions. You may also have to use [legacy logging](#legacy-logging) if you are using functions that do not pass context into the function, for instance [SchemaStateFunc](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/schema#SchemaStateFunc).
```

Comment:
I like this, although maybe it would be best to explicitly call out terraform-plugin-sdk. Would it make sense to move this sentence below the next paragraph so "context" makes more sense? We may also want to adjust the first sentence of the Legacy Logging section to also mention the `context.Context` issue.